packages/instant-meilisearch/src/types/types.ts

```diff
@@ -31,33 +34,49 @@ export type {
   Config as MeilisearchConfig,
 } from 'meilisearch'
 
-export type InstantSearchParams = AlgoliaMultipleQueriesQuery['params']
+export type ApiKeyCallback = () => string
 
-export const enum MatchingStrategies {
-  ALL = 'all',
-  LAST = 'last',
-}
```

Comment:
This should be imported from Meilisearch JS package, and its type fixed there. (https://github.com/meilisearch/meilisearch-js/issues/1588)